2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2047 @@
+are generated every year that will not be re-used after the termination
+of the developer’s position. Part of the problem is the reluctance to
+accept research software engineering as an academic profession that
+results in a lack of incentives to produce high-quality software:
+producing high software quality is resource-demanding, and although the
+San Francisco Declaration on Research Assessment (DORA, 2012)
+demands for a change in the
+academic credit system, many institutions base promotion and
+appointments on traditional metrics like the Hirsch index (Hirsch 2005).
+It is obvious that an extraordinary amount of idealism is required to
+write a sustainable code including documentation and installation
+routines, as well as running infrastructure and giving support to others
+when the resources can be used more profitably by writing scientific
+publications based on fragile prototype software (Bangerth and Heister
+2014; Prins et al. 2015).  
+Thus, one main factor for the poor sustainability of research software
+is the lack of long-term funding for research software engineers
+(RSEs, Richardson and Croucher 2018) who take care of the appropriate
+architecture, organization, implementation, documentation, and community
+interaction for the software, paired with the implementation of measures
+towards making the software sustainable during the development
+process (Brett et al. 2017).  
+In this paper, we describe the state of the practice and current
+challenges for research software sustainability, and suggest measures to
+be taken towards improvements to solve these challenges. The paper is
+the result of a community effort, with work undertaken during two
+workshops and subsequent collaborative work across the larger RSE
+community in Germany: It has been initiated during a half-day workshop
+at the first International Conference for Research Software Engineers in
+Germany (deRSE19) in Potsdam, Germany on 5 June 2019 (de-RSE e.V. -
+Society for Research Software 2019), and continued during a dedicated
+two-day workshop in Berlin, Germany on 7 and 8 November 2019, which was
+funded by Deutsche Forschungsgemeinschaft (DFG). Subsequently, the draft
+produced during the latter event was opened up for review by the German
+RSE community through *de-RSE e.V. - Society for Research Software*.
+
+We mainly focus on the situation of research software and research
+software engineers (RSEs) in Germany, where funding bodies increasingly
+acknowledge the importance and value of sustainable research software
+infrastructures. The German Research Foundation (Deutsche
+Forschungsgemeinschaft, DFG), the largest funding body for basic
+research in Germany, for example, has opened a call for sustainable
+research software development (DFG 2016) at the end of 2016 and a second
+call for quality management in research software (DFG 2019b) in June
+2019. We note that the first call was oversubscribed by a factor of
```

Comment:
```suggestion
2019. The first call was oversubscribed by a factor of
```

Citable source available? If not, just leave it as a statement ("we note that" not needed).

Reply:
Not citeable. Was mentioned in an email from DFG as a response to the submitted letters of intent and proposals.

Reply:
Good enough IMHO. Would still delete "We note that".